TWLight/applications/views.py

```diff
@@ -1393,4 +1398,5 @@ def form_valid(self, form):
             _("Your renewal request has been received. A coordinator will review your request."),
```

Comment:
This message isn't shown in the `My Library` view as expected. I have checked in production and this bug already existed prior to these changes. I'll create a ticket to address it.